README.md

```diff
@@ -1,20 +1,20 @@
 # Description
```

Comment:
Please check other samples on the question of Copyright statements. Shouldn't we add something similar to that sample and related?